src/MakePotCommand.php

```diff
@@ -97,51 +124,71 @@ class MakePotCommand extends WP_CLI_Command {
 	 * [--skip-js]
 	 * : Skips JavaScript string extraction. Useful when this is done in another build step, e.g. through Babel.
 	 *
+	 * [--copyright-holder=<name>]
+	 * : Name to use for the copyright comment in the resulting POT file.
+	 *
+	 * [--package-name=<name>]
+	 * : Name to use for package name in the resulting POT file. Overrides anything found in a plugin or theme.
+	 *
 	 * ## EXAMPLES
 	 *
 	 *     # Create a POT file for the WordPress plugin/theme in the current directory
 	 *     $ wp i18n make-pot . languages/my-plugin.pot
 	 *
 	 * @when before_wp_load
+	 *
+	 * @throws WP_CLI\ExitException
 	 */
 	public function __invoke( $args, $assoc_args ) {
 		$this->handle_arguments( $args, $assoc_args );
+
 		$translations = $this->extract_strings();
+
 		if ( ! $translations ) {
-			WP_CLI::warning( 'No translation found' );
+			WP_CLI::warning( 'No strings found' );
 		}
+
 		$translations_count = count( $translations );
+
 		if ( 1 === $translations_count ) {
 			WP_CLI::debug( sprintf( 'Extracted %d string', $translations_count ), 'make-pot' );
 		} else {
 			WP_CLI::debug( sprintf( 'Extracted %d strings', $translations_count ), 'make-pot' );
 		}
+
 		if ( ! PotGenerator::toFile( $translations, $this->destination ) ) {
 			WP_CLI::error( 'Could not generate a POT file!' );
 		}
+
 		WP_CLI::success( 'POT file successfully generated!' );
 	}
 
 	/**
 	 * Process arguments from command-line in a reusable way.
+	 *
+	 * @throws WP_CLI\ExitException
+	 *
+	 * @param array $args       Command arguments.
+	 * @param array $assoc_args Associative arguments.
 	 */
 	public function handle_arguments( $args, $assoc_args ) {
 		$array_arguments = array( 'headers' );
-		$assoc_args      = \WP_CLI\Utils\parse_shell_arrays( $assoc_args, $array_arguments );
-		$this->source    = realpath( $args[0] );
-		$this->slug      = Utils\get_flag_value( $assoc_args, 'slug', Utils\basename( $this->source ) );
-		$this->skip_js   = Utils\get_flag_value( $assoc_args, 'skip-js', $this->skip_js );
-		$this->headers   = Utils\get_flag_value( $assoc_args, 'headers', $this->headers );
+		$assoc_args      = Utils\parse_shell_arrays( $assoc_args, $array_arguments );
+
+		$this->source           = realpath( $args[0] );
+		$this->slug             = Utils\get_flag_value( $assoc_args, 'slug', Utils\basename( $this->source ) );
+		$this->skip_js          = Utils\get_flag_value( $assoc_args, 'skip-js', $this->skip_js );
+		$this->headers          = Utils\get_flag_value( $assoc_args, 'headers', $this->headers );
+		$this->package_name     = Utils\get_flag_value( $assoc_args, 'package-name', 'Unknown' );
```

Comment:
Shouldn't the package and copyright holder be optional?
If I understand correctly, this will now always be added, but with values of `'Unknown'`.

Reply:
We could either make it optional, change the wording (just „Copyright 2018“ without any author or name) or remove it completely. I‘m not sure if it really serves any purpose.

Also, two command arguments just for this comment is a bit too much I think. Perhaps we could go with a singular argument to manually override that line?

Reply:
Yes, something more generic like `--file-comment` might be preferable.

Reply:
See https://github.com/wp-cli/i18n-command/issues/47#issuecomment-407429435 and #55 for previous discussion about a `--comment` / `--file-comment` argument

Reply:
I think `--comment` could be ambiguous, as it could just as well refer to a translator comment.

Reply:
I added `--file-comment` but left `package-name` as it's needed/useful for the `Project-Id-Version` POT header.